### ISSUE TITLE
CRITICAL: Incorrect detection of app directory under Windows

### ISSUE THREAD

**Issue by tscs37** [ByzantineFailure/BPM-for-Discord#37]
Due to a recent update on PTB the `app-0.0.8` directory changed into `app-0.0.10`

Since both directories remained, the installer script seems to sort them according to the first character and installed BPMforDiscord into the `app-0.0.8` directory and not the `app-0.0.10` directory.

Deleting the `app-0.0.8` directory solved the problem.

**Comment by ByzantineFailure** [comment]
The logic of the PTB installer under windows is the following (which is actually identical to the main installer, just it points to the PTB client folder for looking):

``` javascript
            var folder = _(contents)
                .filter(file => file.indexOf('app-') > -1)
                //Sort by version number, multiple app version folders can exist
                .map(dir => {
                    var version = dir.split('-')[1];
                    var splitVersion = version.split('.');
                    return {
                        name: dir,
                        major: splitVersion[0],
                        minor: splitVersion[1],
                        bugfix: splitVersion[2]
                    };
                })
                .sortBy(["major", "minor", "bugfix"])
                .reverse()    
                .first().name;
```

I believe you are correct, which means I should update the logic for each version to be:
`parseInt(splitVersion[i])`

Good catch, this will go out next release.

Also, I should _really_ take out `reverse().first()` and just use `last()`.  Derp.

**Comment by ByzantineFailure** [comment]
NOTE:  This affects regular Discord too, not just the PTB

**Comment by ByzantineFailure** [comment]
I am also going to amend the PTB installer instructions to note this for the current release.

**Comment by ByzantineFailure** [comment]
http://stackoverflow.com/questions/19313164/sort-items-in-array-by-more-then-one-field-with-lodash/24111473#comment44810920_24111473
The installer uses lodash ^3.10.0

`sortByAll` was merged into `sortBy` in v4.0.0.

... Are you fucking for real with me here.  `parseInt` is necessary, but that alone doesn't fix it, you need to either upgrade to `^4.0.0`  or move to `sortByAll` inside a chained set.

Because the entire distinction is complete horseshit, I am just going to upgrade the lodash version out of irritation.

**Comment by ByzantineFailure** [comment]
I did a bunch of other stuff too, but fixed with this commit fa3aced471875c31172d8852cced8f111cf1502d
